Azure/azure-cli
# [Extensions] Enable CLI extensions to include packages in the 'azure' namespace

**Description**  

Extensions currently cannot modules in the `azure` namespace, which prevents us building extensions to support preview SDKs that are not yet part of the CLI.

This change extends the submodule import path for both `azure` and `azure.mgmt` modules when adding an extension to `sys.path`. This allows extensions to include submodules that _do not_ exist in the CLI's base install (if they do exist, they'll be preferred).

**Testing Guide**  

Create an extension with a dependency on, say, [azure-storage-file-share](https://pypi.org/project/azure-storage-file-share/) and include `import azure.storage.fileshare`. 

Add the extension and try to invoke its command. It will fail without this fix, and succeed with it.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
